content/kubeone/main/guides/cis-benchmarking/cis1.8-kubeone1.8-k8s1.29/_index.en.md

```diff
@@ -524,7 +520,9 @@ _The issue is under investigation to provide a fix in a future KubeOne release_
 ---
 #### 4.2.7: Ensure that the --hostname-override argument is not set (Manual)
 
-**Result:** 🟢 Pass
+**Result:** 🔴 Fail
+
+_The issue is under investigation to provide a fix in a future KubeOne release_
```

Comment:
On what cloud provider did you test this originally?

Reply:
It was GCP